iree/vm/BUILD

```diff
@@ -212,6 +212,13 @@ cc_library(
 #     ],
 # )
 
+iree_cmake_extra_content(
```

Comment:
(not here, but in the future) would be nice to have explicit helpers for this - like iree_cmake_if("${IREE_BUILD_COMPILER} AND ${IREE_BUILD_TESTS}") / iree_cmake_endif()